taxonomies/categories.txt

```diff
@@ -41004,7 +41014,7 @@ bg:Ориз Басмати
 de:Basmati Reis
 es:Arroces Basmati, Arroz Basmati
 fi:basmatiriisit
-fr:Riz Basmati, Basmati, Riz thaï cru, Riz basmati cru
```

Comment:
Maybe you can put "riz thai cru" as a synonym of "fr:Riz thaï, Riz Jasmin"

Reply:
Indeed, I've added it!